code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -685,3 +747,169 @@
+	var/mob/living/carbon/xenomorph/xeno_owner = owner
+	xeno_owner.UnarmedAttack(living_target)
+	step_away(living_target, xeno_owner, 1, 3)
+	xeno_owner.face_atom(living_target)
+
+// ***************************************
+// *********** Phase Out
+// ***************************************
+
+/datum/action/ability/xeno_action/stealth/phaseout
+	name = "Phase Out"
+	action_icon_state = "hunter_invisibility"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Become fully invisible for 6 seconds, or until damaged. Attacking does not break invisibility."
+	ability_cost = 10
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_TOGGLE_PHASEOUT,
+	)
+	cooldown_duration = 6 SECONDS
+	stealth_duration = 6 SECONDS
+	disable_on_signals = list(
+		COMSIG_LIVING_IGNITED,
+		COMSIG_LIVING_ADD_VENTCRAWL,
+	)
+	stealth_flags = DIS_POUNCE_SLASH
+	sneak_attack_armor_pen = 30
+
+// i just realized, we need to disable processing, this will inherently just amke handle_stealth not do anything
+///Updates or cancels stealth
+/datum/action/ability/xeno_action/stealth/phaseout/handle_stealth()
+	var/mob/living/carbon/xenomorph/xenoowner = owner
+	xenoowner.alpha = HUNTER_STEALTH_STILL_ALPHA * stealth_alpha_multiplier // instant full stealth regardless of movement.
+
+// ***************************************
+// *********** Death Mark
+// ***************************************
+/datum/action/ability/activable/xeno/hunter_mark/assassin
+	name = "Death Mark"
+	action_icon_state = "death_mark"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Psionically disturb a creature for 15 seconds, allowing you to deal a stronger sneak attack. They will know you are coming for them."
+	ability_cost = 50
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_HUNTER_MARK,
+	)
+	cooldown_duration = 30 SECONDS
+	require_los = FALSE
+
+	///Duration for the mark.
+	var/timeout = 15 SECONDS
+	///Charge-up duration of the mark where you need to stay still for it to apply.
+	var/chargeup = 2 SECONDS
+
+
+/datum/action/ability/activable/xeno/hunter_mark/assassin/can_use_ability(atom/A, silent = FALSE, override_flags)
+	var/mob/living/carbon/xenomorph/X = owner
+	if(require_los)
+		if(!line_of_sight(X, A)) //Need line of sight.
+			if(!silent)
+				to_chat(X, span_xenowarning("We require line of sight to mark them!"))
+			return FALSE
+	. = ..()
+
+/datum/action/ability/activable/xeno/hunter_mark/assassin/use_ability(atom/A)
+	var/mob/living/carbon/xenomorph/X = owner
+	if(!do_after(X, chargeup, IGNORE_TARGET_LOC_CHANGE, A, BUSY_ICON_HOSTILE, NONE, PROGRESS_GENERIC))
+		return
+
+	to_chat(X, span_xenodanger("We will be able to maintain the mark for [timeout / 10] seconds."))
+	addtimer(CALLBACK(src, PROC_REF(unset_target)), timeout)
+
+	playsound(marked_target, 'sound/effects/alien/new_larva.ogg', 50, 0, 1)
+	to_chat(marked_target, span_highdanger("You feel uneasy."))
```

Comment:
this should be after the parent call, since it could still fail in parent yet give the mob this message.